planet/specs.py

```diff
@@ -111,15 +111,18 @@ def get_item_types(product_bundle=None):
     '''If given product bundle, get specific item types supported by Orders
     API. Otherwise, get all item types supported by Orders API.'''
     spec = _get_product_bundle_spec()
-    if product_bundle:
-        item_types = spec['bundles'][product_bundle]['assets'].keys()
-    else:
-        product_bundle = get_product_bundles()
-        all_item_types = []
-        for bundle in product_bundle:
-            all_item_types += [*spec['bundles'][bundle]['assets'].keys()]
-        item_types = set(all_item_types)
-    return item_types
+    try:
+        if product_bundle:
+            item_types = spec['bundles'][product_bundle]['assets'].keys()
+        else:
+            product_bundle = get_product_bundles()
+            all_item_types = []
+            for bundle in product_bundle:
+                all_item_types += [*spec['bundles'][bundle]['assets'].keys()]
+            item_types = set(all_item_types)
+        return item_types
```

Comment:
An alternative Pythonic approach:

```suggestion
        return set(itertools.chain.from_iterable(spec['bundles'][bundle]['assets'].keys() for bundle in get_product_bundles()))
```

Reply:
@kevinlacaille let's not make a one-line function for `spec['bundles'][product_bundle]['assets']`. I believe a Bundle class would be more effective. Imagine usage like `return set(itertools.chain.from_iterable(bundle.assets for bundle in get_product_bundles())`.

The advantage of `itertools.chain.from_iterable`, IMO, is that it tells me clearly (as reader) what the intent is. Whereas I have to deduce the intent of the more verbose code.